extensions/adobe/experience/campaign/offer.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/offer",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Offer",
+  "type": "object",
+  "description":
+    "This represents the instance of an offer displayed in a message.",
+  "definitions": {
+    "offer": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/offer/id": {
+          "title": "Offer ID",
+          "type": "integer",
+          "description": "Identifier of the offer.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/offer/name": {
+          "title": "Offer Name",
+          "type": "string",
+          "description": "Name of the offer."
+        },
+        "https://ns.adobe.com/experience/campaign/offer/activityId": {
+          "title": "Activity ID",
+          "type": "integer",
+          "description":
+            "Identifier of the offer activity in the Offer Service.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/offer/activityName": {
+          "title": "Activity Name",
+          "type": "string",
+          "description": "Name of the offer activity in the Offer Service."
+        },
+        "https://ns.adobe.com/experience/campaign/offer/placementId": {
+          "title": "Placement ID",
+          "type": "integer",
+          "description": "Identifier of the placement of the offer.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/offer/placementName": {
+          "title": "Placement Name",
+          "type": "string",
+          "description": "Name of the placement of the offer."
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/offerplacement"
```

Comment:
This ref does not exist. It is called `offer` above.

Reply:
:+1: